lib/metalsmith.js

```diff
@@ -121,22 +123,28 @@ module.exports = metalsmith(path.resolve(__dirname, '../'))
     }
   }))
 
-  .use(include({
-    directories: {
-      assets: [
-        path.join('**/', paths.govukfrontend, 'govuk/assets/*')
-      ],
-      'assets/fonts': [
-        path.join('**/', paths.govukfrontend, 'govuk/assets/fonts/*')
-      ],
-      'assets/images': [
-        path.join('**/', paths.govukfrontend, 'govuk/assets/images/*')
-      ],
-      'javascripts/vendor': [
-        path.join('**/', paths.iframeresizer, 'js/*')
-      ]
-    }
-  }))
+  .use((files, metalsmith, done) => {
+    const copyAssets = (pattern, options) => glob.sync(pattern, options).forEach((asset) => {
+      const input = path.join(options.cwd, asset)
+      const output = path.join(options.dest, asset)
+
+      files[output] = {
+        contents: readFileSync(input)
+      }
+    })
+
+    copyAssets('{fonts,images}/*', {
+      cwd: path.join(paths.nodemodules, 'govuk-frontend/govuk/assets'),
+      dest: 'assets'
+    })
+
+    copyAssets('**/{ie8.polyfils,iframeResizer.contentWindow}.@(map|min.js)', {
```

Comment:
@domoscargin This should do the job versus the old `'js/*'`

Matches only these ones:

```mjs
'ie8.polyfils.map',
'ie8.polyfils.min.js',
'iframeResizer.contentWindow.map',
'iframeResizer.contentWindow.min.js'
```

Reply:
I hate that they've misspelt polyfills...

Reply:
You passed the test